<launch>
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
    </node>
    
    <include file="$(find yolov2_ros)/launch/includes/yolo_server.launch" />

    <node pkg="yolov2_ros" name="yolo_predict" type="yolo_predict.py" output="screen">
        <param name="image_topic" type="str" value="/usb_cam/image_raw" />
        <param name="image_type" type="str" value="rgb" />
        <param name="get_object_pos" type="bool" value="false" />
    </node>
</launch>